testsuite/MDAnalysisTests/data/lammps/pairij_coeffs.data

```diff
@@ -0,0 +1,3228 @@
+LAMMPS data file via write_data, version 23 Jun 2022, timestep = 1000
```

Comment:
Would it be possible to gzip or bzip2 this? i.e. is this something we support (I know we do for PDBs and similar, so I assume it should work here too?)

Reply:
@IAlibay I'm sorry, I'm lost a little bit. How can I help at this point?

Reply:
Compress the file(s) with `bzip2` and store it as `testsuite/MDAnalysisTests/data/lammps/pairij_coeffs.data.bz2`. MDAnalysis will just read it the compressed file and we use up less space in the package and on disk.

Reply:
Thank you! Recommitted with a zipped file.